python/mxnet/contrib/onnx/onnx2mx/_op_translations.py

```diff
@@ -714,3 +714,28 @@ def spacetodepth(attrs, inputs, proto_obj):
     new_attrs = translation_utils._fix_attribute_names(attrs, {'blocksize':'block_size'})
 
     return "space_to_depth", new_attrs, inputs
+
+
+def hardmax(attrs, inputs, proto_obj):
+    """Returns batched one-hot vectors."""
+    input_tensor_data = proto_obj.model_metadata.get('input_tensor_data')[0]
+    input_shape = input_tensor_data[1]
+
+    axis = int(attrs.get('axis', 1))
```

Comment:
axis is set to 1 by default? If yes, in next statement else is never executed?

Reply:
should it be -1?

Reply:
As discussed offline:

if axis is negative, the else part of the next statement will set axis to the offset from the start. Example, if length=5 and axis=-2, new axis will be 3.
In case axis is positive, it will maintain the same value for axis.